core/src/focus_tracker.rs

```diff
@@ -157,7 +161,11 @@ impl<'gc> FocusTracker<'gc> {
                         .set_selection(Some(TextSelection::for_range(0, length)), context.gc());
```

Comment:
I don't care too much, but this does more than just update the virtual keyboard. It also sets the text field's selection. So I'm not sure this is the best function name.

Reply:
You're right, I just assumed this was keyboard-related code without looking too deep into this, I removed the method.

I also checked whether we need this at all, and I've found that this behavior is slightly inaccurate, so I added a comment. Perhaps this code could be moved somewhere else in the future.